flask_talisman/talisman.py

```diff
@@ -147,8 +147,8 @@ def init_app(
 
     def _update_local_options(self):
         """Updates view-local options with defaults or specified values."""
-        view_function = flask.current_app.view_functions[
-                flask.request.endpoint]
+        view_function = flask.current_app.view_functions.get(
+                flask.request.endpoint)
 
         view_options = getattr(
             view_function, 'talisman_view_options', {})
```

Comment:
This should be `defaultdict(lambda: _sentinel)`, so that the HTTPS redirect still happens.